modules/SCsub

```diff
@@ -10,7 +10,7 @@ Export("env_modules")
 
 # Header with MODULE_*_ENABLED defines.
 env.CommandNoCache("modules_enabled.gen.h", Value(env.module_list), modules_builders.generate_modules_enabled)
-
+vs_sources = []
```

Comment:
The separation line should be kept for readability.

Reply:
done